### ISSUE TITLE
Module Wolfram broken

### ISSUE THREAD

**Issue by benpro** [sopel-irc/sopel#878]
```
19:59:10 <benpro> .wa 30MB at 1Mbps in hours                                                                                  
19:59:10 <Ika-chan> [WOLFRAM] <html><head><meta http-equiv="content-type" content="text/html = charset=utf-8"><title>404 Not  
                    Found</title></head><body text=#000000 bgcolor=#ffffff><h1>Error: Not Found</h1><h2>The requested URL     
                    <code>/wa/30MB%20at%201Mbps%20in%20hours</code> was not found on this server.</h2><h2></h2></body></html> 
```
I think Wolfram has changed something...

**Comment by dgw** [comment]
Fixed by #867

**Comment by benpro** [comment]
Hmm I searched for and didn't see it... :( Sorry.
